rosco-core/src/main/groovy/com/netflix/spinnaker/rosco/providers/aws/AWSBakeHandler.groovy

```diff
@@ -132,6 +132,11 @@ public class AWSBakeHandler extends CloudProviderBakeHandler {
   }
 
   @Override
+  String getBaseCommand() {
+    return awsBakeryDefaults.templateFile == "aws-chroot.json" ? "sudo" : ""
```

Comment:
Come to think of it, if you go the config route this bit of logic can move to the base `getBaseCommand()`.